test/wallet-http-test.js

```diff
@@ -796,7 +810,7 @@ describe('Wallet HTTP', function() {
       name: name
     }));
 
-    assert.rejects(fn, 'No reveals to redeem.');
```

Comment:
No `await` statement here?

Reply:
Thank you!
rebased to 347e8616f0896302374492099c62046a883367ac